pep-0484.txt

```diff
@@ -1063,6 +1063,54 @@ in turn, to ``collections.abc.Callable``::
       ...
 
 
+The ``NoReturn`` type
+---------------------
+
+The ``typing`` module provides a special type ``NoReturn`` to annotate functions
+that never return normally. For example, a function that unconditionally
+raises an exception::
+
+  from typing import NoReturn
+
+  def stop() -> NoReturn:
+      raise Exception('no way')
```

Comment:
Can you make this RuntimeException? `raise Exception` is such an anti-pattern I don't ever want to see it. :-)